test/spmd/test_xla_sharding.py

```diff
@@ -1100,6 +1100,26 @@ def test_global_mesh(self):
 
     self.assertEqual(id(mesh), id(expected_mesh))
 
+  def test__mark_manual_sharding(self):
```

Comment:
nit. even though it's testing the `_` prefixed api, let's keep it as `test_mark_manual_sharding`